parts/k8s/addons/metrics-server.yaml

```diff
@@ -130,6 +130,17 @@ spec:
         - /metrics-server
         - --kubelet-insecure-tls
         - --kubelet-preferred-address-types=InternalIP
+        - --cert-dir=/tmp
```

Comment:
Are these changes back-compat to v0.3.7? (that's what all the k8s versions besides 1.22 are using)